workshops/interactive_maps/README.md

```diff
@@ -0,0 +1,230 @@
+| Name      | Description |Author|
+| :----: | :----: | :----: |
+| Interactive Maps Using JS | Learn JS and HTML while making a fun interactive map | @LonaDotExe |
```

Comment:
incorrect formatting, pls check other workshops to see correct method. it should be yaml